src/scenes/Partners/Edit/EditOrganizationName.graphql

```diff
@@ -1,12 +1,19 @@
-mutation UpdateOrganizationName($id: ID!, $name: String!) {
-  updateOrganization(input: { organization: { id: $id, name: $name } }) {
+mutation UpdateOrganization($id: ID!, $name: String!, $acronym: String) {
+  updateOrganization(
+    input: { organization: { id: $id, name: $name, acronym: $acronym } }
+  ) {
```

Comment:
Let's use the input object instead of enumerating the fields.
```suggestion
mutation UpdateOrganization($input: UpdateOrganization!) {
  updateOrganization(input: { organization: $input }) {
```
Will you rename the file to match as well?

Reply:
Will do, along with renaming the file